Sources/DotNetGraph/Exceptions/CompilationException.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace DotNetGraph.Exceptions
+{
+    public class CompilationException : Exception
+    {
+        public CompilationException(string compilationError) : base($"'{compilationError}'")
```

Comment:
I'd just pass down the string as is

```suggestion
        public CompilationException(string compilationError) : base(compilationError)
```